workflow/controller/operator_test.go

```diff
@@ -1128,7 +1183,7 @@ func TestStepsRetriesVariable(t *testing.T) {
 		expected = append(expected, fmt.Sprintf("cowsay %d", i))
 	}
 	// ordering not preserved
-	assert.Subset(t, expected, actual)
+	assert.ElementsMatch(t, expected, actual)
```

Comment:
`assert.Subset` should not be used for this case here.

For example, if we have

```
expected = [0, 1, 2, 3, 4, 5]
actual = [0, 0, 0, 0, 0, 0]
```

This case will actually pass since the elements of `actual` is a subset of `expected`.